src/components/UserPortal/ChatRoom/ChatRoom.tsx

```
@@ -669,294 +681,298 @@ export default function chatRoom(props: IChatRoomProps): JSX.Element {
-              <div
-                onClick={() => (chat?.isGroup ? openGroupChatDetails() : null)}
-                className={styles.userDetails}
-              >
-                <p className={styles.title}>{chatTitle}</p>
-                <p className={styles.subtitle}>{chatSubtitle}</p>
+                <div
+                  onClick={() =>
+                    chat?.isGroup ? openGroupChatDetails() : null
+                  }
+                  className={styles.userDetails}
+                >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)